org.gtk.oomox.appdata.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="runtime">
+  <id>org.gtk.oomox</id>
+  <metadata_license>GPLv3</metadata_license>
+  <project_license>GPL-3.0+</project_license>
+  <name>Oomox theme designer</name>
+  <summary>customize icons, xrdb and GTK themes</summary>
+  <description>
+      <p>Graphical application for generating different color variations of Numix and Materia (ex-Flat-Plat) themes (GTK2, GTK3), gnome-colors and ArchDroid icon themes. Have a hack for HiDPI in gtk2.</p>
+      <p>WARNING: Materia and Oomoxify plugins are temporary disabled for Flatpak.</p>
+  </description>
+  <launchable type="desktop-id">org.gtk.oomox.desktop</launchable>
+  <screenshots>
+      <screenshot>
+          <image>https://raw.githubusercontent.com/actionless/oomox/master/screenshot_gui.png</image>
+      </screenshot>
+  </screenshots>
+  <url type="homepage">https://github.com/actionless/oomox</url>
```

Comment:
Needs to have release tags and OARS as well.

Reply:
OARS?

Reply:
Run through the questionnaire here https://hughsie.github.io/oars/ and paste the results into your appdata.